packages/block-library/src/paragraph/edit.js

```diff
@@ -123,6 +123,20 @@ class ParagraphBlock extends Component {
 		}
 	}
 
+	getDisplayedTextColor( backgroundColorClass, textColor ) {
+		// If backgroundColor class is primary, secondary, dark gray, light grary,
+		// then the displayed color of text is '#fff' (white), even if the textColor remains undefined.
+		// If backgroundColor class is white, then the displayed color of text is '#111' (black).
+		// This is a feature to improve accessibility.
+		if ( backgroundColorClass ) {
+			if ( backgroundColorClass === 'has-white-background-color' ) {
+				return '#111';
```

Comment:
Black is encoded as `#000`.

Reply:
Hi, @gziolo Thanks for your comments. I inspect the CSS files.
When I set the background color to primary, secondary, light gray, dark gray, the CSS file is 
![image](https://user-images.githubusercontent.com/24849874/55426196-2fd9e880-5552-11e9-8c14-0f9a9833f781.png)
so the color is '#fff' (white)

When I set the background color to white, the CSS file is
![image](https://user-images.githubusercontent.com/24849874/55426240-46803f80-5552-11e9-892f-8e7fca3e16eb.png)
so the color is '#111' (black)

And you are right, '#000' is also black. But to maintain the consistency, '#111' may be better I think.